pkgs/applications/editors/emacs-modes/melpa-packages.nix

```diff
@@ -248,6 +248,11 @@ env NIXPKGS_ALLOW_BROKEN=1 nix-instantiate --show-trace ../../../../ -A emacsPac
 
           buildInputs = old.buildInputs ++ [ pkgs.tdlib ];
 
+          postPatch = ''
+            substituteInPlace telega-server.el \
+              --replace "telega-directory exec-path" "\"$out/bin\" exec-path"
```

Comment:
This looks flaky. ```emacsWithPackages``` should add "$out/bin" to your exec-path.

Reply:
https://github.com/NixOS/nixpkgs/pull/67833#issuecomment-526842265